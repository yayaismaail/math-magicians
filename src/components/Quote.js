import './Home.css';

const Quote = () => (
  <>
    <h3 className="header">Quote</h3>
    <p className="math-quote">
      As far as the laws of mathematics refer to reality, they are not certain,
      and as far as they are certain,
      they do not refer to reality.
      <strong> ~ Albert Einstein</strong>
    </p>
  </>
);

export default Quote;
