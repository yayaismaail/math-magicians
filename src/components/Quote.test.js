import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

describe('Quote', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
