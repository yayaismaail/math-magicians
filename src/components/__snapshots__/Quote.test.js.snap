// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote matches the snapshot 1`] = `
Array [
  <h3
    className="header"
  >
    Quote
  </h3>,
  <p
    className="math-quote"
  >
    As far as the laws of mathematics refer to reality, they are not certain, and as far as they are certain, they do not refer to reality.
    <strong>
       ~ Albert Einstein
    </strong>
  </p>,
]
`;
